CMakeLists.txt

```diff
@@ -1,142 +1,183 @@
+                 "-DENABLE_TESTS=${ENABLE_TESTS}"
+  )
+
+ExternalProject_Add(nemsio
+  GIT_REPOSITORY "https://github.com/noaa-emc/nceplibs-nemsio"
+  GIT_TAG        "develop"
+  PREFIX         "${CMAKE_BINARY_DIR}/nemsio"
+  CMAKE_ARGS     "-DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}"
+                 "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/nemsio"
+                 "-DENABLE_TESTS=${ENABLE_TESTS}"
+  DEPENDS        bacio w3nco
+  )
+
+ExternalProject_Add(nemsiogfs
+  GIT_REPOSITORY "https://github.com/noaa-emc/nceplibs-nemsiogfs"
+  GIT_TAG        "develop"
+  PREFIX         "${CMAKE_BINARY_DIR}/nemsiogfs"
+  CMAKE_ARGS     "-DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}"
+                 "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/nemsiogfs"
+                 "-DENABLE_TESTS=${ENABLE_TESTS}"
+                 "-DOPENMP=${OPENMP}"
+  DEPENDS        nemsio
+  )
+
+ExternalProject_Add(landsfcutil
+  GIT_REPOSITORY "https://github.com/noaa-emc/nceplibs-landsfcutil"
+  GIT_TAG        "develop"
+  PREFIX         "${CMAKE_BINARY_DIR}/landsfcutil"
+  CMAKE_ARGS     "-DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}"
+                 "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/landsfcutil"
+                 "-DENABLE_TESTS=${ENABLE_TESTS}"
+  DEPENDS        nemsio
+  )
+
+ExternalProject_Add(g2
+  GIT_REPOSITORY "https://github.com/noaa-emc/nceplibs-g2"
+  GIT_TAG        "develop"
+  PREFIX         "${CMAKE_BINARY_DIR}/g2"
+  CMAKE_ARGS     "-DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}"
+                 "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/g2"
+                 "-DENABLE_TESTS=${ENABLE_TESTS}"
+  )
+
+ExternalProject_Add(g2tmpl
+  GIT_REPOSITORY "https://github.com/noaa-emc/nceplibs-g2tmpl"
+  GIT_TAG        "develop"
+  PREFIX         "${CMAKE_BINARY_DIR}/g2tmpl"
+  CMAKE_ARGS     "-DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}"
+                 "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/g2tmpl"
+                 "-DENABLE_TESTS=${ENABLE_TESTS}"
+  )
+
+ExternalProject_Add(w3emc
+  GIT_REPOSITORY "https://github.com/noaa-emc/nceplibs-w3emc"
+  GIT_TAG        "develop"
+  PREFIX         "${CMAKE_BINARY_DIR}/w3emc"
+  CMAKE_ARGS     "-DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}"
+                 "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/w3emc"
+                 "-DENABLE_TESTS=${ENABLE_TESTS}"
+  DEPENDS        nemsio sigio
+  )
+
+if(BUILD_CRTM)
+  ExternalProject_Add(crtm
+    GIT_REPOSITORY "https://github.com/noaa-emc/emc_crtm"
+    GIT_TAG        "develop"
+    PREFIX         "${CMAKE_BINARY_DIR}/crtm"
+    CMAKE_ARGS     "-DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}"
+                   "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/crtm"
+    )
 endif()
 
-# Flag for using static netCDF
-if(STATIC_IS_DEFAULT)
-  set(NETCDF_USE_STATIC_LIBRARIES TRUE)
-else()
-  set(NETCDF_USE_STATIC_LIBRARIES FALSE)
+if(BUILD_POST)
+  ExternalProject_Add(nceppost
+    GIT_REPOSITORY "https://github.com/aerorahul/emc_post"
```

Comment:
Is this by design (pointing to your fork)?

Reply:
Yes. Remember @DusanJovic-NOAA said the post library should be built and not the executable?
Also, this feature branch makes use of imported targets from the NCEPLIBS.  EMC_post does not and cannot handle that, yet. I am about to issue a PR on EMC_post, but I wanted to test that out here with the active users.  Same with UFS_UTILS.  That PR is already out [here](https://github.com/NOAA-EMC/UFS_UTILS/pull/118).

Reply:
This is also the reason (pointing to a user fork) why I did not want to include NCEPpost in this PR.